test/LieGroupsTestSuite.jl/LieGroupsTestSuite.jl

```diff
@@ -390,19 +390,19 @@ end
 Test  `diff_conjugate`
 """
 function test_diff_conjugate(
-    G::AbstractLieGroup, g, h, X; expected=missing, test_mutating::Bool=true
+    G::AbstractLieGroup, g, h, X; expected=missing, test_mutating::Bool=true, atol::Real=0
```

Comment:
```suggestion
    G::AbstractLieGroup, g, h, X; expected=missing, test_mutating::Bool=true, atol::Real=sqrt(eps(real(number_eltype(X)))),
```
0 is rarely a good `atol` to have.